src/main/java/com/ekart/thunderbolt/utils/CommonUtils.java

```diff
@@ -0,0 +1,59 @@
+package com.ekart.thunderbolt.utils;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.server.ResponseStatusException;
+
+public class CommonUtils {
+
+    public static boolean isStringEmpty(boolean includeNullStringAlso, String... parameters){
+        for(String param : parameters){
+            if(param == null || param.length() == 0){
+                return false;
+            } else if(includeNullStringAlso && "null".equals(param)) {
+                return false;
+            }
+        }
+
+        return true;
+    }
+
+    public static Double roundOffDouble(Double rate){
+        return roundOffDouble(rate, null);
+    }
+
+    public static Double roundOffDouble(Double rate, String field){
```

Comment:
I don't see the value field getting used anywhere in the function, only have parameters that are needed.

Reply:
Done